src/_pytest/main.py

```diff
@@ -378,6 +379,7 @@ class Session(nodes.FSCollector):
     Interrupted = Interrupted
     Failed = Failed
     _setupstate = None  # type: SetupState
+    _fixturemanager = None  # type: FixtureManager
```

Comment:
Since `_fixturemanager` is nowhere to be seen in this file, a comment `Set on the session by pytest_sessionstart().`  would be very helpful here.